main.go

```diff
@@ -143,28 +143,54 @@ func registerRHSM(ctx *cli.Context) (error, string) {
 				ctx.StringSlice("activation-key"),
 				ctx.String("server"))
 		} else {
+			var orgs []string
 			if organization != "" {
-				err = registerPassword(username, password, organization, ctx.String("server"))
+				_, err = registerUsernamePassword(username, password, organization, ctx.String("server"))
 			} else {
-				/* TODO: When organization was not specified using CLI option --organization and it is
+				orgs, err = registerUsernamePassword(username, password, "", ctx.String("server"))
+				/* When organization was not specified using CLI option --organization, and it is
 				   required, because user is member of more than one organization, then ask for
-				   the organization. RHSM D-Bus API can provide list of available organizations
-				   in this case. */
-				err = registerPassword(username, password, "", ctx.String("server"))
+				   the organization. */
+				if len(orgs) > 0 {
+					// Stop spinner to be able to display message and ask for organization
+					if isColorful {
+						s.Stop()
+					}
+
+					// Ask for organization and display hint with list of organizations
+					fmt.Printf(
+						"Hint: User \"%s\" is member of following organizations: %v\n",
+						username,
+						strings.Join(orgs, ", "),
+					)
+					scanner := bufio.NewScanner(os.Stdin)
+					fmt.Print("Organization: ")
```

Comment:
I don't have an account to test it with, but would this work?

```suggestion
					if len(orgs) <= 3 {
						fmt.Printf("Organization (%v): ", strings.Join(orgs, ", "))
					} else {
						fmt.Println("Current Organizations")
						for _, org := range orgs {
							fmt.Printf("- %v", org)
						}
						fmt.Printf("Organization: ")
					}
```

Reply:
Why should we print different messages for two/three organizations and for more than 3 organizations? I think that we should be consistent regardless of number of organizations.

Reply:
You are correct. A new-line separated list doesn't make sense if the number of items in the list is really big. But a comma-separated list is equally difficult to view content in that case too. Could we format the list into a tabwriter with 3 or 4 columns?

Reply:
It could probably work for long list of organizations. Question is, how does it look like for small number of organizations (<= 4). I would not print there "Current Organizations:", but I would print there: "Hint: User "ldupont@redhat.com" is member of following organizations:"

Do you want to introduce printing of columns as part of this PR or could it be implemented later in another PR?

Reply:
How about "Available Organizations"?

Reply:
OK